components/IcebreakerGenerator.tsx

```diff
@@ -6,14 +6,12 @@ interface Props {
   currentIcebreaker: Icebreaker;
   actionLabel: string;
   handleGenerateClick: () => void;
-  handleCopyUrlClick: () => void;
 }
 
 export const IcebreakerGenerator = ({
   currentIcebreaker,
   actionLabel,
   handleGenerateClick,
-  handleCopyUrlClick,
```

Comment:
@enriquesanchez, the reason the build fails now is that the `embed` component still provides this property. after removing it from there it's all good

Reply:
🤦 Ah... ok, thank you for showing me! I'm learning 😄